modules/openxr/openxr_api.cpp

```diff
@@ -267,6 +267,16 @@ RID OpenXRAPI::OpenXRSwapChainInfo::get_image() {
 	}
 }
 
+RID OpenXRAPI::OpenXRSwapChainInfo::get_density_map() {
+	OpenXRAPI *openxr_api = OpenXRAPI::get_singleton();
+
+	if (image_acquired && openxr_api && openxr_api->get_graphics_extension()) {
+		return OpenXRAPI::get_singleton()->get_graphics_extension()->get_density_map(swapchain_graphics_data, image_index);
```

Comment:
nit: can just use `openxr_api` instead of `OpenXRAPI::get_singleton()`

Reply:
Thanks, fixed in latest push